library/src/androidTest/java/org/xmtp/android/library/FramesTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.xmtp.android.library
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.xmtp.android.library.frames.ConversationActionInputs
+import org.xmtp.android.library.frames.DmActionInputs
+import org.xmtp.android.library.frames.FrameActionInputs
+import org.xmtp.android.library.frames.FramePostPayload
+import org.xmtp.android.library.frames.FramesClient
+import org.xmtp.android.library.frames.GetMetadataResponse
+import java.net.HttpURLConnection
+import java.net.URL
+
+@RunWith(AndroidJUnit4::class)
+class FramesTest {
+    @Test
+    fun testFramesClient() {
+        val frameUrl = "https://fc-polls-five.vercel.app/polls/01032f47-e976-42ee-9e3d-3aac1324f4b8"
+        val fixtures = fixtures()
+        val aliceClient = fixtures.aliceClient
+
+        val framesClient = FramesClient(xmtpClient = aliceClient)
+        val conversationTopic = "foo"
+        val participantAccountAddresses = listOf("amal", "bola")
```

Comment:
In case you haven't seen we have a new cast https://github.com/jhaaaa/alix-and-bo